work9/index.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const obj = [{
```

Comment:
IMO
I have been told by Mr.Morita that
I should not use “obj” or “array”as a variable name.
Instead, a little more contextual. For example, “data”.

Reply:
Thank you.

I have a question.
Please tell me why you don't use "obj", "array"

Please.


Please confirm that the correction has been completed.

402f0d9

Reply:
Because object and array are widely used.
The content is important.
"obj", "array" are not enough to describe the content. 
It’s my understanding.

Reply:
> "obj"and "array" are not enough to describe the content.

I agree with you.

Reply:
I understand.

No other site used "array" or "obj" for variables.